### PR TITLE
Adding Experiences Page

### PR DESCRIPTION
**Pull request by sowmya9507**

### CODE REVIEW

**Review by israelvictory — COMMENTED** [review]
More awesomeness from you @sowmya9507. Great work! 🥳

**Comment by jhlink** [comment]
I will review this and #117 first thing in the morning, @sowmya9507 !! Wonderful work so far!!

File: server/apps/main/templates/main/experiences_page.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    </style>
+</head>
+
+<body>
+	<nav class="navbar">
+		<a class="navbar-brand" href="#" </a>
+
+			 <a style="color:#000000;"href="#about">About</a>      
+			 <a style="color:#000000;"href="#share">Share</a>      
+			 <a style="color:#000000;"href="#login">Login</a>
```
Comment (jhlink) [inline server/apps/main/templates/main/experiences_page.html:49]: Seems like you're missing a few other buttons, @sowmya9507 . 

Could you add the rest of the buttons as defined on issue #105 ?

**Review by jhlink — CHANGES_REQUESTED** [review]
Great work, @sowmya9507 ! 

There are a couple more changes that need to be made. Afterwards, we can merge this PR. 

Can you link this page to appear when the `View Experiences` button has been clicked in `views.py`?

File: server/apps/main/templates/main/experiences_page.html (hunk 1 above)
Comment (sowmya9507) [reply server/apps/main/templates/main/experiences_page.html:49]: Sure, will do :)

**Comment by sowmya9507** [comment]
@jhlink updated the code with the requested changes :)

**Review by jhlink — CHANGES_REQUESTED** [review]
CleanShot 2021-07-07 at 12 22 01

Hey @sowmya9507 , I'm still unable to view this page when clicking `Public Experiences` on the docker page. 

Is there something I'm missing here? It is a little confusing but `public_experiences` = `experience` page.

In the interim, a new nav button can be added to the main landing page. You can see how Israel did this [here](https://github.com/alan-turing-institute/AutSPACEs/pull/120/files#diff-5beb3587537035b46ab2b8612224ced772c43a7b2a154f204f4d82f74424fed5R203-R211) in his PR, if you need an example.

**Comment by jhlink** [comment]
Changes are good. Merging.

**Pull request merged**